roster2/bundle/src/bundle/card_game_eu.properties

```diff
@@ -0,0 +1,19 @@
+cancel = Cancel
```

Comment:
Hey @PabloSanRoman!
Could you help me with the EU translations please?
Here are the English one's:
```
cancel = Cancel
check_hit_info = Correct! That was {0}!
check_miss_info = Oups, that was wrong! That sutudent\u0027s name was {0}.
check = Check
close = Close
error_loading = The game {0} could not be loaded.
game_name = Name That Face
game_over_info = Congratulations! You know your students pretty well! If you want to practice \
more, reset the game and you can start from the beginning.
no_student_info = There are no students on this site, no names to learn.
progress = You have learned {0} of {1} student names
reroll_user = Reroll student
reset_game_confirm = Are you sure you want to reset the game?
reset_game_info = By resetting the game, wall the learning progress for the sutdents of this site \
will be reset. This might also affect the progress on other sites that share the same students.
reset_game = Reset the game
user_image_alt = Photo of student that you need to guess
user_name_select_placeholder = Enter student name...
whats_the_user_name = What is the name of this student?
```
Greetings,
Markus

Reply:
Hello @stetsche , here is the translation: 

cancel = Utzi
check_hit_info = Ongi! Hau zen!: {0}
check_miss_info = Okertu zara. Ikaslearen izena hau zen: {0}.
check = Egiaztatu
close = Itxi
error_loading = "{0}" jokoa ezin izan da kargatu. 
game_name = Nor da nor?
game_over_info = Zorionak! Oso ongi ezagutzen dituzu zure ikasleak! Gehiago praktikatu nahi baduzu, jokoa berrabiarazi eta hasieratik has zaitezke.
no_student_info = Ez dago ikaslerik gune honetan, izenik ikasteko ez dago.
progress = {1} ikasleen izenetik {0} ikasi duzu
reroll_user = Beste ikasle bat nahastea
reset_game_confirm = Ziur zaude partida berrabiarazi nahi duzula?
reset_game_info = Partida berrabiaraztean gune honetako ikasleen ikasitakoaren aurrerapen guztia galduko da. Berrabiarazteak eragina izan dezake ikasleek partekatzen dituzten beste gune batzuetan ere.
reset_game = Partida berrabiarazi
sound_enable = Gaitu soinu efektuak
sound_disable = Desgaitu soinu efektuak 
user_image_alt = Asmatu behar duzun ikaslearen argazkia
user_name_select_placeholder = Sartu ikaslearen izena... 
whats_the_user_name = Zein da ikaslearen izena?

Reply:
Thank you very much @PabloSanRoman ! I've pushed the translations into the card game